kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/linux/util/LinuxProcessUtil.java

```diff
@@ -268,6 +268,48 @@ public static int getPid(String command, String[] tokens) throws Exception {
             }
         }
     }
+    
+    public static int getPid(int pid) throws Exception {
```

Comment:
wouldn't be better to have this method named as isProcessAlive(int pid) or something related?